api/types/src/derives.rs

```diff
@@ -40,7 +40,7 @@ impl_poem_type!(
     "string",
     (
         example = Some(serde_json::Value::String(
-            "0x000000000000000088fbd33f54e1126269769780feb24480428179f552e2313fbe571b72e62a1ca1"
```

Comment:
I would probably put a comment to not delete the space (before someone does that later)